.github/ISSUE_TEMPLATE/technical-support.md

```diff
@@ -0,0 +1,77 @@
+name: Technical Support Request
+description: File a technical support request
+title: "[Tech Support]: "
+labels: ["technical support"]
+assignees:
+  - straithe
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for taking the time to fill out this technical support request form! Please note that technical support requests can expect a response time of two weeks.  
+  - type: dropdown
+    id: troubleshooting documentation
+    attributes: 
+      label: Have you read the HackRF [troubleshooting documentation](https://hackrf.readthedocs.io/en/latest/troubleshooting.html)?
+      options:
+        - yes
+        - no
+    validations:
+      required: true
+  - type: textarea
+    id: expected outcome
+    attributes:
+      label: What outcome were you hoping for?
+      placeholder: Please be detailed in what you expected to happen
+    validations:
+      required: true
+  - type: textarea
+    id: actual outcome
+    attributes:
+      label: What outcome actually happened? 
+      placeholder: Please be detailed in what did happen
+    validations:
+      required: true
+  - type: checkboxes
+    id: operating systems
+    attributes:
+      label: What operating systems are you seeing the problem on?
+      multiple: true
+      options: 
+        - Ubuntu
```

Comment:
I suggest to change this to `Linux` rather than listing only a single distro.

Reply:
Done!